django_mongodb_backend/base.py

```diff
@@ -325,7 +336,11 @@ def cursor(self):
 
     def get_database_version(self):
         """Return a tuple of the database's version."""
-        return tuple(self.connection.server_info()["versionArray"])
+        # TODO: Remove this workaround and replace with
+        # `tuple(self.connection.server_info()["versionArray"])` when the minimum
+        # supported version of pymongocrypt is >= 1.14.2 and PYTHON-5429 is resolved.
+        # See: https://jira.mongodb.org/browse/PYTHON-5429
+        return tuple(self.connection.admin.command("buildInfo")["versionArray"])
```

Comment:
It's fixed in pymongocrypt 1.15.0 on Aug 5 (1.14.1 was the last 1.14.x release). I think we probably don't need to worry about someone pinning an old version of pymongocrypt though, so we can leave this unchanged, okay?